package.json

```diff
@@ -28,13 +28,13 @@
     "*.js"
   ],
   "scripts": {
-    "pretest": "eslint .",
+    "pretest": "eslint --fix .",
```

Comment:
I found this helpful during development and thought others might so left it in. Can remove if there's disagreement.